content/2016_10_16_how-to-create-a-blog-post.md

```diff
@@ -0,0 +1,48 @@
+Title: How to create a blog post
```

Comment:
I would assume that Pelican uses this metadata somewhere when it generates the HTML, so it probably makes sense to title case it.